File: tests/fixtures/go_project/inventory/inventory.go
// Package inventory — fixture for tests.
package inventory

import "go_project/pricing"

// CheckStock checks stock and returns price.
func CheckStock(itemID int, qty int) float64 {
	return pricing.CalculatePrice(itemID, qty)
}

// ReserveItems reserves items in inventory.
func ReserveItems(itemID int, qty int) bool {
	return qty > 0
}

File: tests/fixtures/rust_project/src/inventory.rs
/// Inventory module — fixture for tests.
use crate::pricing;

/// Check stock and return price.
pub fn check_stock(item_id: u32, qty: u32) -> f64 {
    pricing::calculate_price(item_id, qty)
}

/// Reserve items in inventory.
pub fn reserve_items(item_id: u32, qty: u32) -> bool {
    qty > 0
}
